modules/schematics/collection.json

```diff
@@ -2,52 +2,59 @@
   "extends": ["@schematics/angular"],
   "schematics": {
     "action": {
-      "aliases": [ "a" ],
+      "aliases": ["a"],
       "factory": "./src/action",
       "schema": "./src/action/schema.json",
       "description": "Add store actions"
     },
 
     "container": {
-      "aliases": [ "co" ],
+      "aliases": ["co"],
       "factory": "./src/container",
       "schema": "./src/container/schema.json",
       "description": "Add store container component"
     },
 
     "effect": {
-      "aliases": [ "ef" ],
+      "aliases": ["ef"],
       "factory": "./src/effect",
       "schema": "./src/effect/schema.json",
       "description": "Add side effect class"
     },
 
     "entity": {
-      "aliases": [ "en" ],
+      "aliases": ["en"],
       "factory": "./src/entity",
       "schema": "./src/entity/schema.json",
       "description": "Add entity state"
     },
 
     "feature": {
-      "aliases": [ "f" ],
+      "aliases": ["f"],
       "factory": "./src/feature",
       "schema": "./src/feature/schema.json",
       "description": "Add feature state"
     },
 
     "reducer": {
-      "aliases": [ "r" ],
+      "aliases": ["r"],
       "factory": "./src/reducer",
       "schema": "./src/reducer/schema.json",
       "description": "Add state reducer"
     },
 
     "store": {
-      "aliases": [ "st" ],
+      "aliases": ["st"],
       "factory": "./src/store",
       "schema": "./src/store/schema.json",
-      "description": "Adds initial setup for state managment"
+      "description": "Adds initial setup for state management"
+    },
+
+    "ng-add": {
+      "aliases": ["init"],
+      "factory": "./src/ng-add",
+      "schema": "./src/ng-add/schema.json",
+      "description": "Adds initial setup for Ngrx"
```

Comment:
What about using the following description here?

"Installs the NgRx schematics package"